eradiate/scenes/measure/_core.py

```diff
@@ -510,7 +509,10 @@ def to_dataset(self, aggregate_spps: bool = False) -> xr.Dataset:
                 result["raw"]
                 .weighted(weights)
                 .mean(dim="sensor_id")
-                .expand_dims({"sensor_id": [sensor_id_prefix]}, axis=1)
+                .expand_dims(
+                    {"sensor_id": [int(sensor_id_prefix.split("ms")[1])]},
```

Comment:
This is the place in the code, that I was talking about. Here I expect the measure id to be of the form "something_ms0" and I split the string at "ms" to get the numerical part of the ID. I do this because I cannot tell `xr.combine_by_coords()` to use the alphanumerical sorting method you added. 

I could instead compute this number further up in the code. Basically at the beginning of the loop and perform a check: If `sensor_id_prefix.split("ms")[1]`  is not an empty string, use it and cast it to int, otherwise just use the `sensor_id_prefix` as is. What do you think?